aspnetcore/blazor/hybrid/developer-tools.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Use browser developer tools with ASP.NET Core Blazor Hybrid
+author: guardrex
+description: Learn how to use browser developer tools with Blazor Hybrid apps.
+monikerRange: '>= aspnetcore-6.0'
+ms.author: riande
+ms.custom: "mvc"
+ms.date: 03/23/2022
+no-loc: ["Blazor Hybrid", Home, Privacy, Kestrel, appsettings.json, "ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
```

Comment:
Are these meta keywords? Can we add `.NET MAUI`?

Reply:
MT doesn't run on Blazor docs node. It's all HT ("human translation") here. However, it's possible that some or all of the topics would become MT at some point. Yes, I think it's a good idea to add it. I'll do that, but I'll do it on a separate PR. I'll note it on my tracking issue so that I don't lose this.

Reply:
Oh, sorry ... I missed the ❓ ...

> Are these meta keywords?

No, they're words/phrases to avoid for localization. .NET MAUI doesn't translate, but it would be safe to add it to the list, and I've made a note to do so.

Reply:
Let's add `Mac Catalyst` as well, I've seen that unintentionally localized in the past.

Reply:
Got it ... I added "Mac Catalyst" to the UE tracking issue. It won't get lost.